test/TorchSharpTest/TestDisposeScopesStatisticsTensor.cs

```diff
@@ -73,26 +73,5 @@ public void DisposingScopeAfterDetachingDoesNothing()
             AssertTensorCounts(0, 0, 1, 0, 0, 1, 1);
             AssertTotalsCounts(0, 0, 1, 0, 0, 1, 1);
         }
-
```

Comment:
Why remove the tests?

Reply:
Because the other tests that were added in this PR are more comprehensive, and test in the same way. I created those two as a placeholder to remind me what was wrong and how to recreate it.